Source/JavaScriptCore/JavaScriptCore.xcodeproj/project.pbxproj

```diff
@@ -13549,6 +13592,38 @@
 			};
 			name = Production;
 		};
+		44F93E092AE71F5600FFA37C /* Debug */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = 44F93DFD2AE71EBD00FFA37C /* libJavaScriptCore.xcconfig */;
+			buildSettings = {
+				PRODUCT_NAME = JavaScriptCore;
```

Comment:
nit: You should be able to delete this PRODUCT_NAME setting and the four below (making `buildSettings` an empty dictionary). Xcode adds these by default but the definition is already in xcconfigs.

Reply:
Done!